<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectWindow</class>
 <widget class="QDialog" name="CreateProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>222</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create new project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/imgs/rtree_transparent.png</normaloff>../../Desktop/imgs/rtree_transparent.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QLabel" name="cash_size_label">
     <property name="text">
      <string>Cache size(nodes):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="project_name"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="project_name_label">
     <property name="text">
      <string>Project name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="child_nodes_max_label">
     <property name="text">
      <string>Child nodes maximum:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dimension_label">
     <property name="text">
      <string>Dimension:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="child_nodes_min_label">
     <property name="text">
      <string>Child nodes minimum:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="dimension">
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="create_button">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>362</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>362</width>
       <height>24</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="erase_trashhold_label">
     <property name="text">
      <string>Erased nodes threshold:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="child_nodes_min"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="child_nodes_max"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="cache_size"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="erase_trashhold"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
